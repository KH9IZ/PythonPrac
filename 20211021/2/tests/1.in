:mamaamacriminal t sin(t)/cos(t)==tan(t)
mamaamacriminal 0
:wat c e**log(c) 
wat 9
mamaamacriminal 1
quit tiuq
