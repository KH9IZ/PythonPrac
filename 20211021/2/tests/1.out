True
9.0
False
3
